release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -621,6 +621,91 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
+
+[id=ocp-4-11-monitoring-better-manage-promql-queries-in-web-console]
+==== Create, browse, and manage PromQL queries more easily in the web console
+The Query Browser on the *Observe* -> *Metrics* page of the {product-title} web console adds various enhancements to improve your ability to create, browse, and manage PromQL queries.
+For example, administrators can now duplicate existing queries and use autocomplete suggestions when building and editing queries.
+
+[id=ocp-4-11-monitoring-doubled-scrape-interval-for-servicemonitors-single-node]
+==== Doubled the scrape interval for ServiceMonitors in single node deployments
+The scrape interval has been doubled for all Cluster Monitoring Operator (CMO) controlled ServiceMonitors on single-node {product-title} deployments. 
+The maximum interval is now two minutes.
+
+[id=ocp-4-11-monitoring-create-alerting-rules-based-on-platform-metrics]
+==== Create alerting rules based on platform monitoring metrics (Technology Preview)
+This release introduces a Technology Preview feature in which administrators can create alerting rules based on existing platform monitoring metrics.
+This feature helps administrators to more quickly and easily create new alerting rules specific to their environments.
+
+[id=ocp-4-11-monitoring-add-cluster-id-labels-for-remote-write]
+==== Add cluster ID labels to remote write storage
+You can now add cluster ID labels to metrics being sent to remote write storage.
+You can then query these labels to identify the source cluster for a metric and distinguish that metric data from similar metric data sent by other clusters.
+
+[id=ocp-4-11-monitoring-query-metrics-using-federation-endpoint]
+==== Query metrics using the federation endpoint for user workload monitoring
+You can now use the Prometheus `/federate` endpoint to scrape user-defined metrics from a network location outside the cluster.
+Before this release, you could only access the federation endpoint to scrape metrics in default platform monitoring.
+
+[id=ocp-4-11-monitoring-enable-body-size-limit-metrics-scraping]
+==== Enable body size limit for metrics scraping for default platform monitoring
+You can now set the `enforcedBodySizeLimit` config map option for default platform monitoring to enable a body size limit on metrics scraping.
+This setting triggers the new `PrometheusScrapeBodySizeLimitHit` alert when at least one Prometheus scrape target replies with a response body larger than the configured `enforcedBodySizeLimit`.
+The setting can limit the impact that a malicious target can have on both the Prometheus component and on the cluster as a whole.
+
+[id=ocp-4-11-monitoring-confgure-retention-size-for-metrics-storage]
+==== Configure retention size settings for metrics storage
+You can now configure the maximum amount of disk space reserved for retained metrics storage for both default platform monitoring and user-workload monitoring. 
+Before this release, you could not configure this setting.
```

Comment:
See my earlier comment about ending the section with the formula, `Before this release, you could not...` 

I like this content, and you seem to end roughly half of your section with it. Why not end all the sections with this formula? Or avoid it all together? 

I don't think it's bad to keep them where they are, but right now, I don't see a clear reason why some sections include a statement, `Before this release, you could not...` and others don't.

Reply:
Thanks for pointing that out. I consider adding this wording to be a judgement call on a per item basis based on the feature and whether or not a system limitation or restriction has been removed or expanded by a new feature. In this case and the other places I've added it, it makes sense to me to leave it in. So I'm not going to add it to every item that doesn't have it.